### PR TITLE
RAC-5335_optimize_os_install_smoke

### PR DESCRIPTION
**Pull request by gpaulos**
Resubmit of optimized FIT OS installs.
Optimized by not running stack init routine on OS install, runs SKU pack installer, then OBM settings instead. This will reduce execution time for OS installs.

@hohene @johren @derrickostertag

### CODE REVIEW

**Review by jimturnquist — APPROVED** [review]

**Comment by gpaulos** [comment]
test this please

**Comment by gpaulos** [comment]
test this please

**Comment by gpaulos** [comment]
test this please

**Comment by gpaulos** [comment]
test this please

File: jobs/FunctionTest/FunctionTest.groovy [hunk 1]
```diff
@@ -87,11 +87,12 @@ def functionTest(String test_name, String label_name, String TEST_GROUP, Boolean
                         "TEST_TYPE=$test_type",
                         "TEST_STACK=$test_stack",
                         "EXTRA_HW=$extra_hw",
+                        "LABEL=$label_name",
                         "KEEP_FAILURE_ENV=${env.KEEP_FAILURE_ENV}",
                         "KEEP_MINUTES=${env.KEEP_MINUTES}"]
                     ){
                         try{
-                            timeout(60){
+                            timeout(90){
```
Comment (panpan0000) [inline jobs/FunctionTest/FunctionTest.groovy]: Oops..  will FIT run longer OS install than CIT?

**Comment by gpaulos** [comment]
test this please
\

File: jobs/FunctionTest/FunctionTest.groovy (hunk 1 above)
Comment (gpaulos) [reply jobs/FunctionTest/FunctionTest.groovy]: I should change that back. These optimizations bring the execution time down to 45 minutes.

**Comment by gpaulos** [comment]
test this please

**Comment by gpaulos** [comment]
test this please

**Comment by gpaulos** [comment]
test this please

**Comment by gpaulos** [comment]
test this please

**Comment by gpaulos** [comment]
test this please

**Comment by gpaulos** [comment]
Closing PR unmerged. New Docker deployments are not compatible with this algorithm. Need to revise.

**Pull request closed without merging**